GeeksCoreLibrary/Modules/Ftps/Extensions/EncryptionModesExtensions.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using GeeksCoreLibrary.Modules.Ftps.Enums;
+using FluentFTP;
+
+namespace GeeksCoreLibrary.Modules.Ftps.Extensions;
+
+public static class EncryptionModesExtensions
+{
+    public static FtpEncryptionMode ConvertToFtpsEncryptionMode(this EncryptionModes encryptionMode)
```

Comment:
Graag deze functie herschrijven zodat de waardes 1 op 1 omgezet worden aangezien de namen van de enum waardes exact hetzelfde zijn, bijvoorbeeld zo:
```c#
if (!Enum.TryParse(encryptionMode.ToString("G"), out FtpEncryptionMode ftpEncryptionMode)
{
    throw new NotImplementedException($"Encryption mode '{encryptionMode}' is not recognized or not supported.");
}

return ftpEncryptionMode;
```

Reply:
Code is aangepast en getest om bovenstaande te gebruiken.